packages/cardpay-sdk/sdk/utils/conversions.ts

```diff
@@ -35,9 +31,16 @@ export async function gasPriceInToken(provider: JsonRpcProvider, tokenAddress: s
       Accept: 'application/json',
     },
   });
-
   let gasPriceInNativeTokenInWei = new BN((await gasStationResponse.json()).standard);
 
+  // We use the wrapped native token address because the native token doesn't have an address in Uniswap.
+  // The price of the wrapped native token, such as WETH, is the same as the price of the native token.
+  let wrappedNativeToken = getAddressByNetwork('wrappedNativeToken', network);
+  if (tokenAddress === wrappedNativeToken) {
```

Comment:
I added this line to avoid errors when trying to convert wrapped native token using uniswap sdk.